website/blog/2022-07-26-configuring-grants.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Configuring Grants"
```

Comment:
Also idk why I went with the word Natively for all of these it may be that "Grants as Configs" is better descriptor so

"Writing grants as configs in dbt Core v1.2 for fun and DRYness" or a little less fun but
"Permissioning strategies for your project: grants as configs in dbt Core v1.2"
"Updating our permissioning guidelines: grants as configs in dbt Core v1.2"